frontend/index.html

```diff
@@ -7,10 +7,15 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
     <link rel="stylesheet" href="./styles/index.css">
+    <link rel="stylesheet" href="./styles/leaderboard.css">
     
     <title>Typing Game</title>
 </head>
 <body>
+  <nav>
+      <button id="start-button">Start Game</button>
+      <button id="leaderboard-toggle-button">Toggle Leaderboard</button>
```

Comment:
This is the change that broke a index.js:  `document.getElementById('leaderboard-button').classList.add('hidden');` Make sure if you are changing the name of an id that you change it in all the files where it is referenced!

Reply:
Start Game should be hidden until the user logs in, otherwise the game fails to initialize (no user id)

Reply:
Oops, great point. Thanks! I'll go ahead and change this now